boundary_layer_default_plugin/config/operators/gcp_cloud_function_invoke.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Etsy Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+# see: https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/cloud/operators/functions/index.html
+# and: https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/operators/functions.py
+
+name: gcp_cloud_function_invoke
+operator_class: CloudFunctionInvokeFunctionOperator
+operator_class_module: airflow.providers.google.cloud.operators.functions
```

Comment:
I didn't see any other examples of using `airflow.providers.google.cloud...`. AFAIK it should be supported, just want to make sure that I'm not missing something...

Thanks!

Reply:
Hey @peleyal , we use boundary-layer with airflow version 1.10.3. I'd like to check out this PR and make sure it has no adverse effects on our end since this operator doesn't yet exist there.

Reply:
Sure! Do you have any way to support later versions? Should we for example provide another plugin in this repo for all operators after 1.10.3?

Reply:
Thanks for adding https://github.com/etsy/boundary-layer/issues/114#issuecomment-840243184. I'm just curious now, what happen if you add this operator but don't use it. Can it still break you guys?

Reply:
It probably won't break us but it will likely confuse the community and our users as well if they try to use it. I'd prefer we not add it for now.

Reply:
OK. Thanks for all you details and help!